0000-Proportional-Service-Provider Rewards

```diff
@@ -0,0 +1,63 @@
+# HIP XX: Proportional Service Provider Rewards
+- Author(s): @zer0tweets, @meowshka (Nova Labs, Inc.), @KeithRettig (Discord @KeithR)
+- Start Date: 2023-05-29
+- Category: Economic, Technical
+- Original HIP PR: #nnn
+- Tracking Issue: #nnn
+- Voting Requirements: veMOBILE Holders
+
+## Summary
+This HIP proposes to make modifications to how Service Providers are eligible for rewards from the Service Provider MOBILE Reward bucket.
+
+## Motivation
+The primary motivation is to prevent a scenario in which a Service Provider is rewarded the full 10% MOBILE reward bucket (provided HIP79 passes) while burning little to no Data Credits.
+
+## Stakeholders
+Any and all Service Providers of the Helium MOBILE Network.
+
+## Detailed Explanation
+HIP53 specifies an overall rewards bucket for Service Providers but does not go into specifics of how rewards should be calculated based on the amount of data being offloaded. To prevent a scenario in which a Service Provider is rewarded the full 10% MOBILE reward bucket (provided HIP79 passes) while burning little to no Data Credits (DC), we propose to follow a similar usage-based approach as outlined in HIP10 for the IoT Network.
+We propose that Service Providers are rewarded up to 1:1 in MOBILE tokens for the amount of DC burned during a reward period, similar to the approach in HIP10.  If the Service Providers collectively burn more DC than the equivalent amount of MOBILE tokens in the Service Provider reward bucket, each Service Provider will be rewarded proportionally to its share of DC burn.  If the Service Providers collectively burn less DC than the equivalent amount of MOBILE tokens, the remainder of the Service Provider bucket will not be minted.
+To illustrate, here are two scenarios:
+Value of DCs burned for data traffic is less than the Service Provider reward bucket (10% of MOBILE emissions).
+
+DCs burned by SP        100,000,000
+Price of DC             $0.00001
+Price of MOBILE*        $0.0001
+DCs burned in MOBILE**  10,000,000
+10% SP bucket           500,000,000 MOBILE
+SP rewards              10,000,000 MOBILE
+
+
+Value of DCs burned for data traffic exceeds the Service Provider reward bucket (10% of MOBILE emissions).
+
+DCs burned by SP        100,000,000,000
+Price of DC             $0.00001
+Price of MOBILE*        $0.0001
+DCs burned in MOBILE**  10B
+10% SP bucket           500,000,000 MOBILE
+SP rewards              500,000,000 MOBILE
+
+
+* The price chosen is an example, not the actual price. The actual price will be determined by the Price Oracle on the Solana blockchain during the rewards calculation time.
+** Amount of MOBILE tokens with the same value as the value of the DCs burned by SP: (10,000 DC * $0.00001 (fixed price of DC) / $0.0001 (example price of MOBILE))
+
+While approval of this approach is probably not necessary for Helium Mobile to start offloading data to the Helium Mobile Network, it does modify a policy that is important for Helium Mobile to function.
```

Comment:
Can you explain this further? Why is it important for Helium Mobile, the service provider, to function? Give the reader the context/background.